asciidoctorj-arquillian-extension/build.gradle

```diff
@@ -0,0 +1,28 @@
+dependencies {
+  compile project(path: ':asciidoctorj')
+  compile project(path: ':asciidoctorj-test-support', configuration: 'tests')
+  compile "org.jboss.arquillian.container:arquillian-container-spi:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-test-spi:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-impl-base:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-test-impl-base:$arquillianVersion"
+}
+
+jar.dependsOn project(':asciidoctorj').testClasses
+
+jar {
+  from project(':asciidoctorj').sourceSets.test.output
+  include 'org/asciidoctor/arquillian/**/*'
```

Comment:
That's because the core tests use the Arquillian extension and the Arquillian extension uses the core main classes. 
If I pulled these classes into an own module I had a cyclic dependency.

The only way I see at the moment to resolve this would be to split into core into an API and an Impl part. 



> Am 06.06.2015 um 12:01 schrieb Frank Becker <notifications@github.com>:
> 
> In asciidoctorj-arquillian-extension/build.gradle:
> 
> > @@ -0,0 +1,28 @@
> > +dependencies {
> > +  compile project(path: ':asciidoctorj')
> > +  compile project(path: ':asciidoctorj-test-support', configuration: 'tests')
> > +  compile "org.jboss.arquillian.container:arquillian-container-spi:$arquillianVersion"
> > +  compile "org.jboss.arquillian.container:arquillian-container-test-spi:$arquillianVersion"
> > +  compile "org.jboss.arquillian.container:arquillian-container-impl-base:$arquillianVersion"
> > +  compile "org.jboss.arquillian.container:arquillian-container-test-impl-base:$arquillianVersion"
> > +}
> > +
> > +jar.dependsOn project(':asciidoctorj').testClasses
> > +
> > +jar {
> > +  from project(':asciidoctorj').sourceSets.test.output
> > +  include 'org/asciidoctor/arquillian/**/*'
> Why don't you move the arquillian specific classes from core to this module?
> You wouldn't need the dependency to the core module anymore.
> 
> —
> Reply to this email directly or view it on GitHub.
>